src/python/WMComponent/WorkQueueManager/WorkQueueManagerWorkPoller.py

```diff
@@ -61,11 +63,16 @@ def passRetrieveCondition(self):
         False if the component should skip pulling work this cycle.
         
         For now, it only checks whether the agent is in drain mode or
-        if the condor schedd is overloaded.
+        if the condor schedd is overloaded or
+        MAX_JOBS_PER_OWNER is reached or if the condor schedd is overloaded.
         """
+
         passCond = True
+        myThread = threading.currentThread()
         if isDrainMode(self.config):
```

Comment:
Just to avoid us checking all these constraints, how about we add one info (or warning) log per condition? Then we can remove this log line I think:
https://github.com/ticoann/WMCore/blob/a2f3245d72b595e040bcbe2760ef69c9ba55e903/src/python/WMComponent/WorkQueueManager/WorkQueueManagerWorkPoller.py#L94